### ISSUE TITLE
Abnormal circulating YKL 40 concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6114]
**Preferred term label:** Abnormal circulating YKL 40 concentration

**Synonyms** Abnormal circulating Chitinase-3-like protein 1 (CHI3L1)

**Definition (free text, please give PubMed ID)** Concentration of YKL 40 in the blood is outside the normal range., PMID: 18182077

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased circulating YKL 40 concentration


**Synonyms** Decreased circulating Chitinase-3-like protein 1 (CHI3L1)


**Definition (free text, please give PubMed ID)**  Concentration of YKL 40 in the blood is below the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating YKL 40 concentration


**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**


**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Increased circulating YKL 40 concentration


**Synonyms** Increased circulating Chitinase-3-like protein 1 (CHI3L1)


**Definition (free text, please give PubMed ID)**  Concentration of YKL 40 in the blood is above the normal range., PMID: 18182077

**Parent term (use hpo.jax.org/app)** Abnormal circulating YKL 40 concentration


**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**


**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
https://www.ncbi.nlm.nih.gov/gene/1116
I think YKL40 is an out of date symbol for this gene/protein
chitinase 3-like 1

**Comment by pnrobinson** [comment]
There is no relevant 'reduced' phenotype. 
Therefore
Label: Elevated circulating CHI3L1 level
syn: Elevated circulating YKL40 level
syn:  Elevated circulating chitinase 3-like 1 level
def: Increased concentration of chitinase 3-like 1 (CHI3L1) in the blood circulation.
comment:  chitinase 3-like 1 (CHI3L1; previously known as YKL-40) is a glycoprotein with homology to the chitinase protein family, but without chitinase activity. Elevations of serum CHI3L1 can be measured by ELISA and are related to some kinds of bone disease.
PMID:18182077
PMID:7577068

added as HP:0033103
